crates/revm/src/context/inner_evm_context.rs

```diff
@@ -29,6 +29,8 @@ pub struct InnerEvmContext<DB: Database> {
     pub db: DB,
     /// Error that happened during execution.
     pub error: Result<(), EVMError<DB::Error>>,
+    /// EIP-7702 Authorization list of accounts that needs to be cleared.
+    pub valid_authorizations: Vec<Address>,
```

Comment:
i think this doesn't need to be a vec, per the spec only 1 authorization is ever used per 7702 tx (the first valid one), at least as far as i understand it, and there is currently no way to do a 7702 call from a contract.